packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -100,6 +102,60 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody> {
       ],
     );
   }
+
+  Widget debuggerPanes() {
+    return LayoutBuilder(
+      builder: (context, constraints) {
+        final paneWidth = constraints.maxWidth;
+        return FlexSplitColumn(
+          totalHeight: constraints.maxHeight,
+          initialFractions: const [0.25, 0.25, 0.25, 0.25],
+          minSizes: const [0.0, 0.0, 0.0, 0.0],
+          headers: [
+            _debuggerPaneHeader(callStackTitle, paneWidth),
```

Comment:
seems odd we have to pass in the pane width to the headers

Reply:
turns out we don't need it - removed.